openai/openai_test.go

```diff
@@ -90,73 +91,82 @@ func TestChatMiddleware(t *testing.T) {
 					}
```

Comment:
if `image_url` above on line 87 is the only place we use `imageURL`, let's just inline it there